offix-web/public/stylesheets/styles.css

```diff
@@ -0,0 +1 @@
40px;margin-bottom:100px}.admin-center{width:85%;max-width:1200px}.admin-center .positive{border:2px solid black}.offix{font-size:7.750em;font-family:'Montserrat', 'Open Sans', arial;color:#5083BF;margin:0px}.offix span{color:#fff;font-weight:500}.title{margin:40px;font-size:2.5em}.error{font-size:1.5em;margin:1em;color:red}.form-group{padding:5px 0px}.input-group-addon{padding:6px 12px;font-size:.875em;color:#888;background-color:transparent;border:0px}.form-control{border:0px;border-radius:0;font-size:1em;height:auto;border-bottom:.063em solid #888;background-color:transparent;color:white}.form-control:focus{box-shadow:none;border-bottom:.063em solid white}.login,.register{display:inline-block;vertical-align:text-bottom;width:auto !important;margin:0px 10px;padding:11px 44px;font-size:.688em;line-height:2em;letter-spacing:.08em;text-transform:uppercase;transition:color .2s, background-color .2s, border .2s}.login{color:black;background-color:#ddd}.login:hover{background-color:#999}.register{color:white;background-color:transparent;border:.063em solid #555}.register:hover{color:#ccc;border:.063em solid #ccc}a{color:#00B7FF;cursor:pointer}hr{border-top:.063em #aaa solid}table{margin:0px auto;height:100%;width:100%;color:#ccc}table #table-titles{text-transform:uppercase}th,td{padding:15px 0px;border-left:.125em black solid;border-right:.125em black solid}tr.alternating:nth-child(even){background-color:#2a2a2a}.toggle-button{padding:0}.toggle-button input{width:100%;height:100%;transition:background-color .2s;background-color:transparent;border:0px}.toggle-button input:focus{outline-color:transparent}.remove-button{margin-left:10px;height:100%;font-size:1.25em;vertical-align:middle;color:#bb3d30;transition:color .2s;cursor:pointer}.remove-button:hover{color:red}.remove-button form{position:absolute;width:100%;height:100%;top:0;left:0}.remove-button input{width:100%;height:100%;background-color:transparent;border:none}tr.positive input:hover{background-color:#158948}tr.negative input:hover{background-color:#953b32}.mac{line-height:1.5em}td.positive,tr.positive,tr.negative{color:white}td.positive,tr.positive{background-color:#259958}td.negative,tr.negative{background-color:#bb3d30}@media only screen and (min-device-width: 320px) and (max-device-width: 1100px) and (-webkit-min-device-pixel-ratio: 2){body{font-size:32px;line-height:32px}td{padding:40px 0px}.offix{font-size:6em}.input-group-addon{font-size:1.2em}.form-control{font-size:1.8em}.register{margin:40px;font-size:1em;border:.15em solid #555;padding:20px 60px;letter-spacing:0}.login{margin:40px;font-size:1em;border:.1em solid #ddd;padding:20px 60px;letter-spacing:0}.center{width:95%;max-width:none;padding:40px 30px 0px 30px}.admin-center{width:95%;max-width:none}.back{display:inline-block;font-size:1.3em;margin-top:10px;margin-bottom:4em}}@media only screen and (min-device-width: 700px) and (max-device-width: 1100px) and (-webkit-min-device-pixel-ratio: 2){body{font-size:18px;line-height:18px}}
```

Comment:
build artifacts shouldn't be checked in. could you remove this and add it to the gitignore?